client/state/current-user/gravatar-status/actions.js

```diff
@@ -34,20 +41,40 @@ export function uploadGravatar( file, bearerToken, email ) {
 					dispatch( {
 						type: GRAVATAR_UPLOAD_REQUEST_SUCCESS
 					} );
+					dispatch( recordTracksEvent( 'calypso_edit_gravatar_upload_success' ) );
 				} );
 				fileReader.readAsDataURL( file );
 			} )
 			.catch( () => {
 				dispatch( {
 					type: GRAVATAR_UPLOAD_REQUEST_FAILURE
 				} );
+				dispatch( recordTracksEvent( 'calypso_edit_gravatar_upload_failure' ) );
+				dispatch( bumpStat( 'calypso_gravatar_update_error', 'unsuccessful_http_response' ) );
 			} );
 	};
 }
 
-export function receiveGravatarImageFailed( errorMessage ) {
-	return {
-		type: GRAVATAR_RECEIVE_IMAGE_FAILURE,
-		errorMessage
+export function receiveGravatarImageFailed( { errorMessage, statName } ) {
+	return dispatch => {
+		dispatch( recordTracksEvent( 'calypso_edit_gravatar_file_recieve_failure' ) );
+		dispatch( bumpStat( 'calypso_gravatar_update_error', statName ) );
+		dispatch( {
+			type: GRAVATAR_RECEIVE_IMAGE_FAILURE,
+			errorMessage,
+		} );
+	};
+}
+
+export function clickButton( { isVerified } ) {
```

Comment:
👍 Moved this and `receiveImage` to `<EditGravatar />` in 0777aad6e1c61ec2d600da56d0f99a9437aaff4a